lib/oci_utils/metadata.py

```diff
@@ -274,39 +270,30 @@ def _name_convert_underscore(self, meta):
 
         return new_meta
 
-    def _post_process(self, metadata):
+    def _post_process(self):
         """
         Due to the different attribute names from the instance metadata
         service and from the SDK, we need to convert the names from the SDK
         to the names from the instance metadata service.
 
-        Parameters
-        ----------
-        metadata: dict
-           The metadata.
-
-        Returns
-        -------
-            dict
-                The converted metadata.
         """
         # merge extendedMetadata into metadata
-        if 'instance' in metadata and metadata['instance'] is not None:
-            if 'metadata' in metadata['instance']:
-                if 'extendedMetadata' in metadata['instance']:
-                    v = metadata['instance'].pop('extendedMetadata')
-                    metadata['instance']['metadata'].update(v)
+        if 'instance' in self._metadata and self._metadata['instance'] is not None:
+            if 'metadata' in self._metadata['instance']:
+                if 'extendedMetadata' in self._metadata['instance']:
+                    v = self._metadata['instance'].pop('extendedMetadata')
+                    self._metadata['instance']['metadata'].update(v)
             else:
-                if 'extendedMetadata' in metadata['instance']:
-                    v = metadata.pop('extendedMetadata')
-                    metadata['metadata'] = v
+                if 'extendedMetadata' in self._metadata['instance']:
+                    v = self._metadata.pop('extendedMetadata')
+                    self._metadata['metadata'] = v
 
         # change vnic's id to vnicId
-        if 'vnics' in metadata:
-            for i in range(len(metadata['vnics'])):
-                v = metadata['vnics'][i].pop('id')
-                metadata['vnics'][i]['vnicId'] = v
-        return metadata
+        if 'vnics' in self._metadata:
+            for i in range(len(self._metadata['vnics'])):
+                v = self._metadata['vnics'][i].pop('id')
+                self._metadata['vnics'][i]['vnicId'] = v
+
```

Comment:
did this ever work?

Reply:
I hope so. you should ask this to the owner of these lines.